cmd/exchanger/main.go

```diff
@@ -9,53 +9,52 @@ import (
 
 	"github.com/ethanjmarchand/exchanger/internal/controller"
 	"github.com/ethanjmarchand/exchanger/internal/currency"
-	"github.com/go-chi/chi"
-	"github.com/go-chi/chi/middleware"
 	"github.com/joho/godotenv"
 )
 
-// TODO: Test this funtion.
-// Mixing concerns. This should only load an Env variable. Should be returning a string, or config struct.
-func loadEnvKey() (string, error) {
+// loadEnv simple grabs our env variables from our .env file, and returns a config.
+func loadEnv() (*currency.Config, error) {
+	config := currency.Config{}
```

Comment:
since this is just a struct with a single field, I'd be tempted to return a string here. If you do want to keep it a struct, you don't need to initialise it until we return since there is only one place we assign to it